content/en/docs/_index.md

```diff
@@ -1,20 +1,42 @@
 
 ---
-title: "Tekton Documentation"
+title: "Welcome to Tekton"
 linkTitle: "Documentation"
 weight: 20
 menu:
   main:
     weight: 20
 ---
 
-Tekton is a cloud native continuous integration and delivery (CI/CD)
-solution. It allows developers to build, test, and deploy across cloud
-providers and on-premise systems.
+Tekton is an open-source cloud native CI/CD (Continuous Integration and
```

Comment:
Typically the first time you introduce the name of a product, system, or feature, you write out the full name of the feature first, then include the acronym in parentheses, and subsequently only use the acronym. Not sure this change is necessary here.